### PR TITLE
grub2: efi support

### PR DESCRIPTION
**Pull request by Ratler**

### CODE REVIEW

**Comment by lunar-linux-b01** [comment]
Build finished. 
SUCCESS!

**Comment by lunar-linux-b01** [comment]
SUCCESS!

**Comment by Florin65** [comment]
The --disable-werror appears twice in the BUILD file.

**Comment by Florin65** [comment]
In fact the --enable-werror was replaced with --enable-strict-compilation option for the actual gcc compiler version.

**Comment by Ratler** [comment]
This PR is required in order to provide an additional EFI boot loader for the lunar-iso, see lunar-linux/lunar-iso#11

**Comment by lunar-linux-b01** [comment]
Build finished. 
SUCCESS!

**Comment by lunar-linux-b01** [comment]
SUCCESS!

File: system/grub2/BUILD [hunk 1]
```diff
@@ -1,15 +1,52 @@
...
+  _EFI_ARCH="x86_64"
+  _EFI_EMU="--enable-efiemu"
+else
+  _EFI_ARCH="i386"
+  _EFI_EMU="--disable-efiemu"
+fi &&
+
+OPTS+=" --prefix=/usr \
+        --bindir=/usr/bin \
+        --sbindir=/usr/bin
```
Comment (dagbrown) [inline system/grub2/BUILD]: Shouldn't there be a backslash at the end of this line?

**Comment by lunar-linux-b01** [comment]
Build finished. 
SUCCESS!

**Comment by lunar-linux-b01** [comment]
SUCCESS!

**Comment by lunar-linux-b01** [comment]
Build finished. 
SUCCESS!

**Comment by lunar-linux-b01** [comment]
SUCCESS!

**Pull request merged**